gql/resolver_dealpublish.go

```diff
@@ -163,3 +172,41 @@ func (r *resolver) DealPublishNow(ctx context.Context) (bool, error) {
 	r.publisher.ForcePublishPendingDeals()
 	return true, nil
 }
+
+func (r *resolver) PublishPendingDeals(ctx context.Context, args dealsToPublish) (bool, error) {
+	if !r.publisher.ManualPSD() {
+		return false, fmt.Errorf("manual deal publishing is disabled")
+	}
+
```

Comment:
I think it's fine if people want to publish specific pending deals, even when manual psd is not enabled.
I'd just remove this check here.